securedrop_client/api_jobs/uploads.py

```diff
@@ -93,20 +93,21 @@ def call_api(self, api_client: API, session: Session) -> str:
             # Update following draft replies for the same source to reflect the new reply count
             draft_file_counter = draft_reply_db_object.file_counter
             draft_timestamp = draft_reply_db_object.timestamp
+
             update_draft_replies(
                 session,
                 source.id,
                 draft_timestamp,
                 draft_file_counter,
                 new_file_counter,
-                commit=False,
```

Comment:
Why change this to commit _before_ adding the reply to replies table, bumping the source interaction count, and deleting the draft reply?